build-system/tasks/nailgun.js

```diff
@@ -74,13 +75,17 @@ function maybeReplaceDefaultCompiler() {
  * @param {boolean} detached
  */
 async function startNailgunServer(port, detached) {
+  await maybeGenerateRunner(port);
```

Comment:
Any reason why we generate the nailgun runner even though there's a possibility it might not be needed?

Reply:
This was intentional, because we still invoke the runner without nailgun.

https://github.com/ampproject/amphtml/blob/21c6804018384a03ef13b5f909cc0bd331bffdf6/build-system/compile/closure-compile.js#L97-L101